easybuild/easyconfigs/u/UDA/UDA-2.6.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'UDA'
+version = '2.6.0'
+
+homepage = 'https://github.com/ukaea/UDA'
+description = """Universal Data Access"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/ukaea/UDA/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+checksums = ['9fd8b917a4f6bc0eaadb04cc6b04fb98cebcdee7f5a31ec7bc2604839ca2e395']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('libMemcached', '1.0.18'),
+    ('libtirpc', '1.3.2'),
+]
+
+dependencies = [
+    ('libxml2', '2.9.13'),
+    ('Boost', '1.79.0', '-no_mpi'),
+    # OS dependency should be preferred if the os version is more recent then this version
+    # it's nice to have an up to date openssl for security reasons
+    # ('OpenSSL', '1.1.0e'),
+]
+
+# osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

we can use the `OpenSSL` wrapper module for this:

```suggestion
    ('OpenSSL', '1.1', '', SYSTEM),
]
```